src/atomate2/lobster/schemas.py

```diff
@@ -775,3 +1151,76 @@ def _get_strong_bonds(
                         }
                     )
     return bond_dict
+
+
+def read_saved_json(
```

Comment:
Why do you need this function. Can't you just use `monty.serialization.loadfn`?

Reply:
Is it really that bad? I'd imagine this would only cause an issue if the files are above 1GB.

Reply:
The alternative would be to store these files in an alternative format like hdf5

Reply:
> Is it really that bad? I'd imagine this would only cause an issue if the files are above 1GB.

Yeah, the size usually ends up of JSONS above 1 GB (on average 2-4 GB: just checked a few generated files now) in the uncompressed state when using a larger number of points say 10000 points for writing COHPCAR, COOPCAR, COBICAR, in LOBSTER runs. 

To add to this, the database we constructed had fairly simpler and smaller structures, It will be much larger for cases when moving to complicated and larger structures 

Thus I also added an option to skip storing this larger cohpcar objects in the db

Reply:
We need to definitely have this function `read_saved_json` as you can see below, the memory requirements can be super large to load these files for larger structures.